docs/howto/manage-ssh-keys.md

```diff
@@ -52,7 +52,7 @@ This will add the SSH key to the default user account named 'ubuntu'.
 
 ## View the available SSH keys
 
-To list the currently known SSH keys for the current model, use the `ssh-keys` command. 
+To list the currently known SSH keys for the current model, use the `ssh-keys` command.
```

Comment:
"list the known"? The double up of currently/current is a bit sour.

Reply:
Fixed, thanks!